[Challenge #1] Média dos alunos

DiSousaDev opened the pull request.

Commit:
Finalizando o desafio

Logikoz reviewed (approved):
Boa noite primo! O quê você acha de fazer um plus pra esse desafio? Se você puder ter infinitas notas para fazer essa média, como você faria? Caso também o usuário colocar algum tipo de informação não numérica dentro da sua aplicação, como ela se portaria?

Caso você queira o desafio, tá ai uma boa pedida. Porém caso você não quiser, me sinaliza no post para que eu possa mergear o PR!

Commit:
Medias Infinitas

Utilizei um while e o metodo da Classe Scanner que fica aguardando o valor a ser digitado.
